data-raw/create_adams_data.R

```diff
@@ -122,18 +122,18 @@ run_template <- function(tp) {
 
 
 if (update_pkg) {
-  github_pat <- Sys.getenv("GITHUB_TOKEN") # in case of run through github workflows
```

Comment:
I would uncomment this, it's important to use access token to avoid github access rate issues